File: A/C++/Disk.cpp
#include <string>
#include <iostream>

using namespace std;

// Kelas Disk
class Disk
{
    // Atribut Private
    private:
        string type;
        string capacity;
        int price;

    // Atribut Public
    public:
        // Constructor
        Disk()
        {
        
        }

        // Constructor Input
        Disk(string type, string capacity, int price)
        {
            this->type = type;
            this->capacity = capacity;     
            this->price = price;
        }

        // Metode Setter
        void setType(string type)
        {
            this->type = type;
        }

        void setCapacity(string capacity)
        {
            this->capacity = capacity;
        }

        void setPrice(int price)
        {
            this->price = price;
        }

        // Metode Getter
        string getType()
        {
            return this->type;
        }

        string getCapacity()
        {
            return this->capacity;
        }

        int getPrice()
        {
            return this->price;
        }

        // Metode Print
        void printDisk()
        {
            cout << "===============================" << endl;
            cout << "Hardisk:" << endl;
            cout << "===============================" << endl;
            cout << "Tipe           : " << this->type << endl;
            cout << "Kapasitas      : " << this->capacity << endl;
            cout << "Harga          : " << "Rp" << this->price << endl;
        }

        // Destructor
        ~Disk()
        {

        };
};

File: B/PHP/Disk.php
<?php

// Kelas Disk
class Disk
{
    private $type;
    private $capacity;
    private $price;

    // Constructor
    function __construct($type, $capacity, $price) 
    {
        $this->type = $type;
        $this->capacity = $capacity;
        $this->price = $price;
    }

    // Metode Setter
    function setType($type) 
    {
        $this->type = $type;
    }

    function setCapacity($capacity) 
    {
        $this->capacity = $capacity;
    }

    function setPrice($price) 
    {
        $this->price = $price;
    }

    // Metode Getter
    function getType() 
    {
        return $this->type;
    }

    function getCapacity() 
    {
        return $this->capacity;
    }

    function getPrice() 
    {
        return $this->price;
    }

    // Metode Print
    function printDisk() 
    {
        echo "===============================". "<br/>";
        echo "Disk:". "<br/>";
        echo "===============================". "<br/>";
        echo "Tipe          : ". $this->getType(). "<br/>";
        echo "Kapasitas     : ". $this->getCapacity(). "<br/>";
        echo "Harga         : ". "Rp". $this->getPrice(). "<br/>";
    }
}
?>